hoot-core/src/main/cpp/hoot/core/io/HootApiDbReader.cpp

```diff
@@ -97,7 +97,7 @@ void HootApiDbReader::open(QString urlStr)
     if (_email == "")
     {
       throw HootException("If a map name is specified then the user email must also be specified "
-                          "via: " + emailKey());
+                          "via: " + ConfigOptions().getApiDbEmailKey());
```

Comment:
I saw the `ServiceDbEmptyTagsTest` fail once on another branch once but never saw it again.  I'll be that when you change the code to use the static function it will build.  It seems to be a rare false failure.